src/components/book/book.css

```diff
@@ -29,7 +29,7 @@ article {
 
   /* margin: 0 2.438rem 0 0; */
   opacity: 0.5;
-  font-family: Montserrat, sans-serif;
+  font-family: "Montserrat", sans-serif;
```

Comment:
- [ ] For the best practices, please remove the repeating lines and assign one class attribute for multiple elements with the same rules as here. You can do the same thing for `display: flex` and other repetitive lines.

Reply:
Extra padding from requirements  : IMPORTANT NOTE in case of problems with Zeplin login - please use [the project created by one of Microverse students - Usman Suleiman Babakolo](https://pedantic-beaver-cb923b.netlify.app/#/).